src/connectors/connectorSwUpdates.js

```diff
@@ -70,6 +70,9 @@ export default class ConnectorSwUpdates extends Connector{
 
     subscribe = (input) =>
         new Promise((resolve, reject) => {
+            console.log(this.config.checkForUpdatesAtBoot);
+            console.log(this.config.checkForUpdatesInterval);
+            console.log(this.config.checkForUpdates);

```

Comment:
don't use `console.log` there is a logger you can import from `env.js`

Reply:
actually this is just debug, I'll remove it